tasks/post-to-cmr/index.js

```diff
@@ -54,6 +55,25 @@ async function addMetadataObjects(cmrFiles) {
   return updatedCMRFiles;
 }
 
+/**
+ * Check that each granule to upload contains a CMR Metadata file
+ * Throws error if not
+ *
+ * @param {Array} granules - granules object from input
+ * @param {Array} cmrFiles -  yay
+ */
+function checkForMetadata(granules, cmrFiles) {
+  if (cmrFiles.length === 0) {
+    throw new CMRMetaFileNotFound('No CMR Meta file found.');
+  }
+  const granuleIds = cmrFiles.map((g) => g.granuleId);
```

Comment:
So at this point `cmrFiles` only contains objects referencing the actual CMR metadata files?

Reply:
yeah it's an array objects that are pairs. Each pair is a metadata filename and the granuleId of the granule it comes from.